test/lib/builder/builder.js

```diff
@@ -86,14 +94,14 @@ test("Build application.a", (t) => {
 	return builder.build({
 		tree: applicationATree,
 		destPath,
-		excludedTasks: ["generateComponentPreload", "generateStandaloneAppBundle", "generateVersionInfo"]
+		excludedTasks: ["generateComponentPreload", "generateStandaloneAppBundle", "generateVersionInfo", "generateResourcesJson"]
```

Comment:
"generateResourcesJson" is not default anymore and therefore does not need to be excluded here

Reply:
done